app/Filament/Resources/UserResource/Pages/ListUsers.php

```diff
@@ -133,6 +135,8 @@ public function table(Table $table): Table
                     RestoreBulkAction::make(),
                     AssignLicensesBulkAction::make()
                         ->visible(fn () => auth()->user()->can('create', License::class)),
+                    AssignRolesBulkAction::make()
+                        ->visible(fn () => auth()->user()->can('create', Role::class)),
```

Comment:
Let's change the permission check here to make sure they can `update` `User`

Reply:
Done